### ISSUE TITLE
macos is super scffedu

### ISSUE THREAD

**Issue by frosencoffee** [ppy/osu#11330]
**Describe the bug: ever since november updates, the UI has been practically unusable** 

**Screenshots or videos showing encountered issue: no screenshots available at this time. the issue is that on macos, once in the game in fullscreen mode, you cannot switch between windows, cannot open any other programs and are forced in osu unless you fully quit out of the application. osu lazer also unoptimized. another issue is that you cannot adjust mouse sensitivity, mouse raw input button wont let you click it an ymore, its grayed out. cannot use tablet with osu lazer anymore, lazer recognizes tablet but doesnt recognize user sensitivty area etc.** 

**osu!lazer version: 2020.1225.0** 

**Logs:**
